<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mine Craft Mod Comparer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>361</width>
      <height>801</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <widget class="QLabel" name="mainLabel">
       <property name="text">
        <string>Main</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mainCombo">
       <property name="placeholderText">
        <string>(Select...)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="mainList">
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mainButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>mainList</zorder>
    <zorder>mainLabel</zorder>
    <zorder>mainButton</zorder>
    <zorder>mainCombo</zorder>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>361</width>
      <height>801</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="followerLayout">
     <item>
      <widget class="QLabel" name="followerLabel">
       <property name="text">
        <string>Follower</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="followerCombo">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="placeholderText">
        <string>(Select...)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="followerList">
       <property name="currentRow">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="followerButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>followerList</zorder>
    <zorder>followerButton</zorder>
    <zorder>followerLabel</zorder>
    <zorder>followerCombo</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
